compiler/rustc_hir_pretty/src/lib.rs

```diff
@@ -553,17 +553,7 @@ impl<'a> State<'a> {
             }
             hir::ItemKind::OpaqueTy(opaque_ty) => {
```

Comment:
Is this branch reachable in practice? :thinking: `type T = impl B` gets printed as `type T = /*impl Trait*/` since the RHS is a `hir::TyKind::OpaqueDef`.